tests/modules/sightings/resources/utils.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Project resources utils
+-------------
+"""
+import json
+import config
+from tests import utils as test_utils
+from flask import current_app
+
+
+PATH = '/api/v1/sightings/'
+
+
+def get_transaction_id():
+    return '11111111-1111-1111-1111-111111111111'
+
+
+def prep_tus_dir():
```

Comment:
This doesn't seem particularly sighting specific and would be needed for the encounter testing.... I think?
Move to tests/utils  maybe?

Reply:
agree.

Reply:
yup; will add this to next pr hopefully (call me out if not!) as it will be a bit more testing there....